### ISSUE TITLE
Event Registration with Ubercart (at least try it out)

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/backdrop_events#28]
@mazzech Suggested the possibility of using Ubercart for event registration. 

Some folks might suggest that Ubercart is a bit heavy for this small problem, but I could also argue that events sometimes have other e-commerce needs. 

I think it's worth trying. Personally, I've never used ubercart so I think that this would be an interesting experiment. 

I'm opening this ticket to suggest that someone experiment with Ubercart as the solution and update us on whether or not it works? Does Backdrop have all the components we need? Is Ubercart a viable solution for this problem?

@mazzech suggested this D6 tutorial that might still be helpful. 
https://www.drupaleasy.com/blogs/ultimike/2009/03/event-registration-ubercart

**Comment by mazzech** [comment]
I will check it out anytime soon... event handling is pretty important for me, but the devil is in the detail, e.g. changing registrations, categories etc.

**Comment by stpaultim** [comment]
Noticing that the 7.x-2.x version of [COD for Drupal 7](https://www.drupal.org/project/cod) includes the Commerce module. I assume for handling registration, sponsorships, and other financial transactions.

**Comment by mazzech** [comment]
Yes, I also remember that I noticed this some years ago, did some research on COD. I assume this module is also being used https://www.drupal.org/project/registration_commerce, it extends this one here https://github.com/backdrop-contrib/registration

**Comment by mazzech** [comment]
BTW, the project I did back then failed, and it was also because the lack of experience on my side regarding events. Looking back, I regret that I did not choose CiviCRM, which seems to have a better OOTB event solution
